Bottom: 1a7cac69690186ee75fcb56298dbecd74b7f92c0
Top:    95dd2359033f97ebff23415fc140ccfac5aaf521
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-19 01:37:49 +0000

hid/gtk (GL): I think the polygon renderer works in mask mode now


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 26f7225..74b3db6 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1400,11 +1400,8 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
-  common_fill_pcb_polygon (out->fgGC, &polygon, screen);
+  gui->graphics->fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_FreeContours (&polygon.NoHoles);
-  /* THE GL fill_pcb_polygon doesn't work whilst masking */
-//  gui->fill_pcb_polygon (out->fgGC, &polygon, screen);
-//  gui->graphics->fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
   ghid_set_alpha_mult (out->fgGC, 1.0);
 
   gui->graphics->use_mask (HID_MASK_OFF);
